ui/lib/core/addon/components/secret-list-header-tab.hbs

```diff
@@ -0,0 +1,11 @@
+{{#unless this.dontShowTab}}
+  {{#if this.isEngine}}
+    <LinkTo @route={{@link}}>
```

Comment:
Links are handled a little differently in engines, and I needed to account for that here.